core/src/components/datetime/datetime.ionic.scss

```diff
@@ -216,6 +216,17 @@
 
 // Calendar / Footer / Action Buttons
 // -----------------------------------
+:host .datetime-footer {
+  @include globals.padding(globals.$ion-space-200, globals.$ion-space-0, globals.$ion-space-200, globals.$ion-space-0);
+  @include globals.margin(globals.$ion-space-200, 0, 0, 0);
+  @include globals.border(
+    globals.$ion-border-style-solid globals.$ion-border-size-025 globals.$ion-primitives-neutral-300,
```

Comment:
This color doesn't seem to align with the Figma color. It seems to be `#E0E0E0` while the Figma design uses `#⁨⁨⁨⁨D5D5D5`.

Reply:
You're right dividers color updated